simplified-app-ekirjasto/src/main/assets/txnative/en/txstrings.json

```diff
@@ -232,7 +241,7 @@
          "string" : "%1$s (%2$d of %3$d)"
       },
       "app_name" : {
-         "string" : "E-kirjasto"
+         "string" : "simplified-viewer-epub-readium2"
```

Comment:
This should be changed to "E-kirjasto" in Transifex, the old value keeps coming back from there.

Reply:
So the issue here is that the same string ID is used in multiple modules, which means that the string from one module is overriding the string from the other... Both source strings are set to "E-kirjasto", but somehow this still has the old value in Transifex. Needs more investigation, I'll handle this separate of this PR.